src/backend/src/utils/validation.utils.ts

```diff
@@ -11,23 +12,32 @@ export const nonEmptyString = (validationObject: ValidationChain): ValidationCha
 };
 
 export const isRole = (validationObject: ValidationChain): ValidationChain => {
-  return validationObject.isString().isIn(['APP_ADMIN', 'ADMIN', 'HEAD', 'LEADERSHIP', 'MEMBER', 'GUEST']);
+  return validationObject
+    .isString()
+    .isIn([RoleEnum.ADMIN, RoleEnum.APP_ADMIN, RoleEnum.GUEST, RoleEnum.HEAD, RoleEnum.LEADERSHIP, RoleEnum.MEMBER]);
 };
 
 export const isDate = (validationObject: ValidationChain): ValidationChain => {
   return validationObject.custom((value) => !isNaN(Date.parse(value)));
 };
 
 export const isTaskPriority = (validationObject: ValidationChain): ValidationChain => {
-  return validationObject.isString().isIn(['LOW', 'MEDIUM', 'HIGH']);
+  return validationObject.isString().isIn([TaskPriority.High, TaskPriority.Medium, TaskPriority.Low]);
 };
 
 export const isTaskStatus = (validationObject: ValidationChain): ValidationChain => {
-  return validationObject.isString().isIn(['IN_BACKLOG', 'IN_PROGRESS', 'DONE']);
+  return validationObject.isString().isIn([TaskStatus.DONE, TaskStatus.IN_BACKLOG, TaskStatus.IN_PROGRESS]);
 };
 
 export const isWorkPackageStageOrNone = (validationObject: ValidationChain): ValidationChain => {
-  return validationObject.isString().isIn(['RESEARCH', 'DESIGN', 'MANUFACTURING', 'INTEGRATION', 'NONE']);
+  return validationObject
+    .isString()
+    .isIn([
+      WorkPackageStage.Design,
```

Comment:
samething here but research design manufacuring integration, and then you have to add ‘NONE’ as just a raw string